incubator/patroni/templates/statefulset-patroni.yaml

```diff
@@ -77,7 +83,7 @@ spec:
   - metadata:
       name: pgdata
       annotations:
-        volume.alpha.kubernetes.io/storage-class: default
+        volume.alpha.kubernetes.io/storage-class: {{default "default" .Values.StorageClass }}
```

Comment:
@joar check out the prometheus chart:
https://github.com/kubernetes/charts/blob/master/stable/prometheus/templates/alertmanager-deployment.yaml#L72